packages/components/src/Components/ConditionalFilter/Group.js

```diff
@@ -215,9 +221,13 @@ class Group extends Component {
 
     render() {
         const { isExpanded, filterBy } = this.state;
-        const { groups, items, placeholder, className, selected, isFilterable, isDisabled, onFilter } = this.props;
-
+        const { groups, items, placeholder, className, selected, isFilterable, isDisabled, onFilter, onShowMore } = this.props;
         const filterItems = items || groups;
+        const showMore = {
```

Comment:
You can move that to consts file, since you are not accessing any props etc.

Reply:
I suppose `onShowMore` property will be accessed there